ci/builders/linux_host_engine.json

```diff
@@ -0,0 +1,174 @@
+{
+    "builds": [
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_custom_vars": {
+                "download_android_deps": false
+            },
+            "gn": [
+                "--runtime-mode",
+                "debug",
+                "--unoptimized",
+                "--prebuilt-dart-sdk"
+            ],
+            "name": "host_debug_unopt",
```

Comment:
@zanderso @dnfield I don't think we need to build host_debug_unopt here as it has now its own builder and we don't upload any of its artifacts. am I correct?

Reply:
The `flutter` target isn't built in engine.py https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine/engine.py#770.

For `flutter/sky/packages`, I think after https://github.com/flutter/engine/pull/27240, https://github.com/flutter/engine/blob/main/testing/run_tests.py#L405 is not needed anymore, but with that still around, run_tests.py will expect those files to be there.

As for the exception_shaders, those are generated test data. That data will have to be available, but it isn't necessarily pulled from host_debug_unopt. Instead it comes from the `FLUTTER_BUILD_DIRECTORY` env var: https://github.com/flutter/engine/blob/main/testing/dart/shader_test_file_utils.dart#L10, which is set by run_tests.py: https://github.com/flutter/engine/blob/main/testing/run_tests.py#L140.

Reply:
Removed flutter and flutter/sky/packages. If we remove this builder we can still build and run tests without problems.

Reply:
May need to coordinate this with Dart 3H

Reply:
@athomas @whesse something to discuss in tomorrow's meeting. I'll leave it like this for now.